jitsi_meet/tasks/installation.yml

```diff
@@ -0,0 +1,42 @@
+---
+
+- name: install jitsi-meet related packages
+  package:
+    name: '{{ item }}'
```

Comment:
Use `{{ jitsi_meet_related_packages }}` instead of `{{ item }}` and remove the `with_items`, because this is faster.